tests/expectations/metrics/test_map_metric.py

```diff
@@ -78,7 +78,8 @@ def sqlite_table_for_unexpected_rows_with_index(
             except ValueError:
                 pass
 
-            return sqlite_engine
+            yield sqlite_engine
```

Comment:
is mypy not upset about this? I would expect you'd have to update the return type here to be a Generator (that yields a sqlalchemy.Engine)

Reply:
Good point I didn't catch this and I didn't see any mypy complaints.